lib/capybara/selenium/driver.rb

```diff
@@ -148,6 +148,10 @@ def evaluate_async_script(script, *args)
     unwrap_script_result(result)
   end
 
+  def active_element
+    build_node(browser.switch_to.active_element)
```

Comment:
Technically this changes the behavior of Session#send_keys when using the selenium driver because `build_node` will build instances specialized for the specific browser being used - which may not end up directly calling the Selenium `send_keys` directly - was this intentional? (it may actually be an improvement, just checking)

Reply:
Not intentional. Calling `build_node` was an attempt to transform an instance of `Selenium::WebDriver::Element` into an instance of `Capybara::Node::Element` for the equivalence check in the test suite. Is there a better way or a different abstraction to use?

Reply:
No -- just wanted to make sure you realized that `build_node` would return a ::Capybara::Selenium::ChromeNode instance when using the selenium driver with Chrome, which has some extra code to work around an issue with emoji in send_keys  -- not necessarily a bad thing just different than the previous behavior